Networking/Sources/Networking/Networking.swift

```diff
@@ -1,6 +1,61 @@
+import Foundation
+import Alamofire
 public struct Networking {
```

Comment:
```suggestion

public struct Networking {
```

Line spaces are good for readability